### ISSUE TITLE
Add direct image outputs

### ISSUE THREAD

**Issue by fsimonis** [precice/config-visualizer#18]
The config-visualizer currently outputs dot graphs which need to be rendered by passing them to dot.
Most users just want an output, be it PDF or PNG. Piping into dot is an unnecessary complication.

I propose to make the CLI output-sensitive and render files if the extensions are supported by dot.

```console
$ precice-config-visualizer precice-config.xml
Outputs dot to stdout as before
$ precice-config-visualizer -o config precice-config.xml
Outputs dot as the the extension of the output is unclear
$ precice-config-visualizer -o config.dot precice-config.xml
Outputs dot as before
$ precice-config-visualizer -o config.pdf precice-config.xml
Renders to PDF
$ precice-config-visualizer -o config.png precice-config.xml
Renders to PNG
```

**Comment by MakisH** [comment]
I upvote this, but I would go one step further and make PNG (preferrably) or PDF the default.
If one needs a DOT file, they probably know how to look for it. Most of the time, we just need something ready to see in a common tool.

**Comment by fsimonis** [comment]
Solved in 629f5ceb0934867e949b723c1927e28360802a71

Supports png, pdf, svg, jpeg, jpg, and dot.
I decided to treat stdout and unknown extensions as "output the generated graph".
